sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala

```diff
@@ -467,9 +470,11 @@ abstract class TreeNode[BaseType <: TreeNode[BaseType]] extends Product {
   }
 
   /**
-   * All the nodes that are parts of this node, this is used by subquries.
+   * All the nodes that are private children of current node.
```

Comment:
"Private children" is quite vague. Maybe:

> All the nodes that should be shown as a inner nested tree of this node. For example, this can be used to show sub-queries.